### PR TITLE
DTFS2-5385 GEF Bug - return to maker MIA bug

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Bug: 
Edge case: on MIA -> MIN, if not issuing any unissued facilities, returning to maker unlocked the full application details page

# Changes made:

* Added condition for map summary list for edge case if MIA, submission count > 0, and if changes required
* Added application details parameter for ^
* Locked certain buttons using application details parameter
* Added e2e test for edge case (MIA accepting decision without changing unissued facilities)

### CODE REVIEW

File: gef-ui/server/utils/helpers.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ const userToken = (req) => {
 // Checks to see if an element is an object or not
 const isObject = (el) => typeof el === 'object' && el !== null && !(el instanceof Array);
 
+const isMIAWithoutChangedToIssuedFacilities = (app) =>
+  app.status === CONSTANTS.DEAL_STATUS.CHANGES_REQUIRED && app.submissionType === CONSTANTS.DEAL_SUBMISSION_TYPE.MIA && app.submissionCount > 0;
```
Comment (abhi-markan) [inline gef-ui/server/utils/helpers.js:27]: Good practice to have conditions in new line, to improve readability.

**Review by abhi-markan — CHANGES_REQUESTED** [review]

**Comment by abhi-markan** [comment]
Good work on the bug and resolution.

File: gef-ui/server/utils/helpers.js (hunk 1 above)
Comment (Zainzzkk) [reply gef-ui/server/utils/helpers.js]: done

**Review by abhi-markan — APPROVED** [review]
Nice work

**Review by IgnatG — APPROVED** [review]

**Pull request merged**